hooks/pre_gen_project.py

```diff
@@ -5,16 +5,25 @@
 Web/API Variant
 cookiecutter._web_variant
 cookiecutter._api_variant
+cookiecutter._only_web_variant
+cookiecutter._both_variant
 -----------
 {% if cookiecutter.variant in ['Web only', 'Both'] %}
     {{ cookiecutter.update({ '_web_variant': 'yes' }) }}
 {% endif %}
 
-API Variant
+{% if cookiecutter.variant == 'Web only' %}
+    {{ cookiecutter.update({ '_only_web_variant': 'yes' }) }}
+{% endif %}
```

Comment:
Could we use `_web_variant == yes && _both_variant == no` instead of having this? 💭

Reply:
Also, I think having 3 variants is pretty hard to handle, as it'll bring a lot of conditions in the template so I think we should consider [splitting the template](https://cookiecutter.readthedocs.io/en/stable/advanced/nested_config_files.html)

Reply:
That logic should work, my expectation was trying to simply the logic with one condition - `_only_web_variant` is more private property. But I will apply to logic `_web_variant == yes && _both_variant == no` to reduce one variable.

Reply:
@carryall And about splitting the template, that idea is better. Our other templates already applied that approach. What do you think about removing `only web` variant? Normally, for web project, we will have `only API` and `both only API and web` (e.g., rails-templates)

Reply:
@carryall I fixed in https://github.com/nimblehq/gin-templates/pull/106/commits/65b389d6662b178adef4f418e8d8d81376e7ccda with new logic `_web_variant == yes && _both_variant == no`

Reply:
>What do you think about removing only web variant?
Agree @suho , I updated [the issue](https://github.com/nimblehq/gin-templates/issues/111)